install.py

```diff
@@ -261,6 +260,7 @@ def install_bench(args):
 	extra_vars.update(bench_name=bench_name)
 
 	# Will install ERPNext production setup by default
+	log("Initializing bench using bench_name=%s frappe_branch=%s erpnext_branch=%s ..." % (bench_name, frappe_branch, erpnext_branch))
```

Comment:
```suggestion
	log("Initializing bench {0}:\n\tFrappe Branch: {1}\n\tERPNext Branch: {2}".format(bench_name, frappe_branch, erpnext_branch))
```

Can we have something like this instead? Also, check if `--without-erpnext` or `--without-bench-setup` are passed and show the appropriate statement log here

Reply:
>  Also, check if --without-erpnext or --without-bench-setup are passed and show the appropriate statement log here

Address this too ^